### ISSUE TITLE
Setting default folder for .tex outputs and relative path for images

### ISSUE THREAD

**Issue by psyguy** [raineszm/obsidian-export-to-tex#18]
Since compiling `.tex` file makes a lot of auxiliary files, it is better to keep them in a folder. It is good to set that in plugin options and so that relative paths to images are adapted to that.

For instance, if the exports are "by default" saved in `TeX exports` folder (placed in vault's root) then the relative paths to the attached images in `\includegraphics` should appear as `../attachments/figures/some image.png` instead of `attachments/figures/some image.png`. 

Thanks :)

**Comment by raineszm** [comment]
So the intention is that TeX files will be exported to a folder for that TeX project. If you're going to leave the image files in place then it makes sense to use the absolute path option. If not it's probably best to copy the image files to from your vault to the TeX directory. In that case maybe the default should be to just use the file base name in the includegraphics command i.e. `some image.png`.

Is your use case that you want to keep your tex projects inside the vault?

**Comment by psyguy** [comment]
> Is your use case that you want to keep your tex projects inside the vault?

Yes, indeed, I want to save the `.tex` files within my vault in their designated folder. So my vault looks something like this:

```
📦vault-root
 ┣ 📂notes
 ┃ ┗ 📜note1.md
 ┃ ┗ 📜note2.md
 ┣ 📂attachments
 ┃ ┗  📂figures
 ┃     ┗ 📜some image.png
 ┃     ┗ 📜another image.png
 ┣ 📂TeX exports
 ┃ ┗ 📜note1.tex
 ┃ ┗ 📜note1.aux
 ┃ ┗ 📜note1.pdf
 ┃ ┗ ...
```

**Comment by raineszm** [comment]
Ah, okay. I need to think a little about how I want to implement this, but I should be able to add some combination of options to support this workflow.

**Comment by psyguy** [comment]
Some note here: it's cool if the default export folder is specified in plugin settings. This would be useful when exporting to clipboard.

**Comment by raineszm** [comment]
This is implemented in the newest version [0.2.0](https://github.com/raineszm/obsidian-export-to-tex/tree/0.2.0).
